depends/packages/qt.mk

```diff
@@ -35,6 +35,7 @@ $(package)_config_opts += -no-freetype
 $(package)_config_opts += -no-gif
 $(package)_config_opts += -no-glib
 $(package)_config_opts += -no-icu
+$(package)_config_opts += -no-ico
```

Comment:
For reviewers: We have .ico files in-tree, but they're processed at build-time by windres.